### ISSUE TITLE
Does the uuidvalue from the drift_progres package not have a default converter?

### ISSUE THREAD

**Issue by MathiasCochet** [simolus3/moor#2849]
I have a User table with the primary key an id with the uuid custom type: 
`UuidColumn get id => customType(PgTypes.uuid)();`

When I try to find a user by the Id I get an error: 
```
ERROR - 2024-01-20 14:05:43.619274
GET /user?id=d952eb9f-7ad2-4b1b-b3ce-386735205991
Error thrown by handler.
Invalid argument (value): Unsupported type: Instance of 'UuidValue'
package:drift_postgres/src/pg_database.dart 180:14                           new _BoundArguments.ofDartArgs
package:drift_postgres/src/pg_database.dart 137:36                           _PgDelegate.runSelect
package:drift/src/runtime/executor/helpers/engines.dart 77:19                _BaseExecutor.runSelect.<fn>
package:drift/src/runtime/executor/helpers/engines.dart 57:22                _BaseExecutor._synchronized.<fn>
dart:async                                                                   new Future.sync
package:drift/src/utils/synchronized.dart 17:21                              Lock.synchronized.callBlockAndComplete
package:drift/src/utils/synchronized.dart 23:14                              Lock.synchronized
package:drift/src/runtime/executor/helpers/engines.dart 55:20                _BaseExecutor._synchronized
package:drift/src/runtime/executor/helpers/engines.dart 74:26                _BaseExecutor.runSelect
package:drift/src/runtime/query_builder/statements/select/select.dart 81:16  SimpleSelectStatement._getRaw.<fn>
package:drift/src/runtime/api/connection_user.dart 162:64                    DatabaseConnectionUser.doWhenOpened.<fn>
```

And this is the code I wrote to find the user:
```
 @override
  Future<UserData?> getUserById(UuidValue userId) async {
    logger.d('userId: $userId');
    return (db.user.select()
          ..where((tbl) {
            return tbl.id.equals(userId);
          }))
        .getSingle();
  }
```

**Comment by simolus3** [comment]
Which drift version are you on? I think this should have been fixed in drift 2.15.0.

**Comment by MathiasCochet** [comment]
Ah I am on 2.14.1, I will try 2.15.0! Thanks!
